_pages/en_US/installing-boot9strap-(usm).txt

```diff
@@ -16,8 +16,11 @@ Once the WiFi profile has been injected, we will use SAFE_MODE, which is a recov
 
 These instructions work on USA, Europe, Japan, and Korea region consoles as indicated by the letters U, E, J, or K after the system version.
 
-If your (Right/Left Shoulder), (D-Pad Up) or (A) buttons do not work, you will need to use a [Legacy Method](legacy-methods). For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
-{: .notice--danger}
+If you have an existing Homebrew Entrypoint using BrowserHax 2020, you should follow [BH-Slottool](installing-boot9strap-(BH-Slottool))
+{: .notice--warning}
+
+If you have an existing Homebrew Entrypoint using something other than BrowserHax 2020, you should follow [HBL-Slottool](installing-boot9strap-(HBL-Slottool))
+{: .notice--warning}
```

Comment:
People would only ever get linked to this page through the legacy method, so putting these warnings here would only cause confusion